cgi-bin/json.cgi

```diff
@@ -62,11 +75,7 @@ esac
 
 case "$set" in
   on)
-    if [ ! -z $mins ]; then
-      echo "echo 1 > $RELAY_CTRL" | at now + $mins minute -M -q b
-    else
-      echo 1 > $RELAY_CTRL
-    fi
```

Comment:
Hmm... this looks weird. Probably the cause of the issue in #8. Something might've gone wrong when resolving merge conflicts.